src/main/java/com/example/imagescaler/ImageScalerApplication.java

```diff
@@ -1,15 +1,17 @@
 package com.example.imagescaler;
 
+import com.example.imagescaler.upscale.configuration.UpscaleApiConfig;
 import io.swagger.v3.oas.annotations.OpenAPIDefinition;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 
 @SpringBootApplication
 @OpenAPIDefinition
+@EnableConfigurationProperties(UpscaleApiConfig.Configuration.class)
```

Comment:
You don't need this, it's already here https://github.com/lasha-gamrekelashvili/image-scaler/pull/10/files#diff-68abe65201a1e247c45c01e9576bb8b58f6f276a20f797660a8f296d044c02b7R9